packages/wrangler/src/dev.tsx

```diff
@@ -567,6 +569,12 @@ export async function startApiDev(args: StartDevOptions) {
 		additionalModules,
 	} = await validateDevServerSettings(args, config);
 
+	const nodejsCompatMode = validateNodeCompat(
+		args.nodeCompat ?? config.node_compat ?? false,
+		args.compatibilityFlags ?? config.compatibility_flags,
+		args.noBundle ?? config.no_bundle ?? false
+	);
```

Comment:
nit but these booleans could end up passed in the wrong order – better to make the params an object of named options

Reply:
Good call. I'll update.